pkg/kubelet/token/token_manager.go

```diff
@@ -72,8 +73,9 @@ func NewManager(c clientset.Interface) *Manager {
 			}
 			return tokenRequest, err
 		},
-		cache: make(map[string]*authenticationv1.TokenRequest),
-		clock: clock.RealClock{},
+		cache:     make(map[string]*authenticationv1.TokenRequest),
+		clock:     clock.RealClock{},
+		maxJitter: 10 * time.Second,
```

Comment:
Move to package level const like maxTTL.

Reply:
done